ui/package.json

```diff
@@ -42,6 +42,7 @@
     "react-dropzone": "12.1.0",
     "react-redux": "7.2.8",
     "react-router-dom": "5.3.1",
+    "react-router-dom-v5-compat": "6.3.0",
```

Comment:
This package allows us to migrate parts of the app to React Router v6 without doing it all at once.